### DISCUSSION TITLE
Multiple templates

### DISCUSSION THREAD

**Discussion by jongablop** [xtreamsrl/jupytemplate#10] [Ideas]
Hi all,

Is there any way to have multiple templates to choose from when a new notebook is created?

I love your work!

Thanks,

Jon

**Comment by donlelef** [comment]
Hi @jongablop! 

Thank you for the idea. Unfortunately, at the present moment the feature is not supported, but we may think about adding it. Would you be interested in contributing?

Have a nice beginning of 2021!

**Reply by jongablop** [reply]
Hi @donlelef!

I've never worked on a Jupyter extension, but I'll be ready to help if you are interested in adding that feature.

Happy new year!

**Reply by donlelef** [reply]
Great Thank you very much! I'll be a bit busy for the return from the Christmans break, but I'll let you know as soon as I start working on this, so that we can join forces.

**Reply by donlelef** [reply]
Happy new year as well! Hope that 2021 will pay off the debt 2020 has contracted with the world!

**Reply by jongablop** [reply]
Perfect! you can send me an email when you get down to it.
